modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -2501,6 +2509,47 @@ private Map<String, Serializable> _getValuesFromDatabase(
 		return values;
 	}
 
+	private void _testObjectEntryWithMultiselectPicklist() throws Exception {
+		ObjectEntry objectEntry = _addObjectEntry(
+			HashMapBuilder.<String, Serializable>put(
+				"emailAddressRequired", "john@liferay.com"
+			).put(
+				"listTypeEntryKeyRequired", "listTypeEntryKey1"
+			).put(
+				"multipleListTypeEntriesKey",
+				(Serializable)Collections.singletonList("listTypeEntryKey1")
+			).build());
+
+		Assert.assertNotNull(objectEntry);
+
+		Map<String, Serializable> values = _objectEntryLocalService.getValues(
+			objectEntry.getObjectEntryId());
+
+		Assert.assertEquals(
+			"listTypeEntryKey1", values.get("multipleListTypeEntriesKey"));
+
+		objectEntry = _objectEntryLocalService.updateObjectEntry(
+			TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
+			HashMapBuilder.<String, Serializable>put(
+				"multipleListTypeEntriesKey",
+				(Serializable)Arrays.asList(
+					"listTypeEntryKey1", "listTypeEntryKey2")
+			).build(),
+			ServiceContextTestUtil.getServiceContext());
+
+		Assert.assertNotNull(objectEntry);
+
+		values = _objectEntryLocalService.getValues(
+			objectEntry.getObjectEntryId());
+
+		Assert.assertEquals(
+			"listTypeEntryKey1, listTypeEntryKey2",
+			values.get("multipleListTypeEntriesKey"));
+
+		_objectEntryLocalService.deleteObjectEntry(
+			objectEntry.getObjectEntryId());
+	}
+
```

Comment:
Can you adapt this code and put it under the testUpdateObjectEntry method?

There, you'll have to:

1. Add a new multiselect field here: 

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L188-L249

2. Add new checks here:

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L2012-L2030

and here:

https://github.com/liferay/liferay-portal/blob/e573e6fefc0c22989f3e5c2930f3d4ac67c9539f/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L2077-L2099

And feel free to adapt/create code for the needs you want. Thx

Reply:
copy that

Reply:
The first point I had already done